.travis.yml

```diff
@@ -0,0 +1,35 @@
+# Config file for automatic testing at travis-ci.org
+sudo: true 
+branches:
+  only:
+    - master
+    - test
+language: python
+python:
+  - 3.6
+services:
+  - postgresql
+install: 
+  - pip install -r requirements.txt
+before_script:
+  - psql -c 'create database ether_sql;' -U postgres
+  - psql -c "CREATE USER root WITH PASSWORD 'develop';" -U postgres
+  - psql ether_sql < tests/ether_sql_test.sql
+  - export FLASK_APP="api/api.py"
+  - export FLASK_DEBUG=1
+
+script:
+  - |
+    python api/api.py &
+    APP_PID=$!
+  - curl -s https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip > ngrok.zip
+  - unzip ngrok.zip
+  - ./ngrok http 5000 > /dev/null &
```

Comment:
shouldnt `flask run` be enough to start the server? Why was ngrok needed?

Reply:
`flask run &` should be enough

Reply:
Tried it, doesn't work unless app.run() is removed from api.py, in which case it no longer stays running after a few seconds of timeout. 
ngrok configures the server to a different tunnel so that we can continue to make curl requests on this thread.

Reply:
When the server runs no other command can be made from the same terminal. Using the & to push it to the background doesn't work either as it is run through the app.run() in the api.py. 
Ngrok was the only solution I found upon looking for ways to test Flask API urls